### ISSUE TITLE
Error thrown after updating to Meteor 1.7.0

### ISSUE THREAD

**Issue by emaciel10** [qualialabs/prod-shell#1]
Looks like this line is throwing an error after the update to Meteor 1.7.0
https://github.com/qualialabs/prod-shell/blob/master/server/shell_server.js#L79

Here is the error that gets logged in the console:
```
Compiling undefined with meteor-babel without a cache
[object Object]
     at Object.exports.compile (/Users/eric/.meteor/packages/babel-compiler/.7.1.0.13p6xfm.gnrf++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-babel/index.js:77:13)
     at Object.compile (packages/babel-compiler.js:44:29)
     at Object.compileForShell (packages/ecmascript/ecmascript.js:7:18)
     at Object.createShellClient (packages/qualia:prod-shell/server/shell_server.js:79:53)
     at Object.ensureShellServer (packages/qualia:prod-shell/server/shell_server.js:21:12)
     at main.js (packages/qualia:prod-shell/server/main.js:3:13)
     at fileEvaluate (packages/modules-runtime.js:339:7)
```

I tried updating babel runtime with the command below but still no luck.
```
meteor npm install @babel/runtime@latest
```

**Comment by etyp** [comment]
@veered any idea here? We love the prod shell (started using it after your Meteor night talk) and don't want to abandon it as we upgrade to Meteor 1.7 --- happy to help provide more info or reproduction repo to debug if you'd like.

**Comment by veered** [comment]
I'll take a look this week

**Comment by veered** [comment]
I don't think anything is broken, it still works for me. The error logged to the console is just a warning that the babel compiler isn't using the cache https://github.com/meteor/babel/blob/master/index.js#L74

This warning messages appears to be new to Meteor 1.7. I'll look into suppressing it, but everything should be working as it was before.

**Comment by veered** [comment]
Alright published version `0.0.3` to Atmosphere which has a small change to prevent the warning message from printing.

https://github.com/qualialabs/prod-shell/commit/7dad6d16a230539b5c16ab24da963efc47ac1749#diff-f2aca2455aa035784e37700476c2104dR85
